src/components/App.js

```diff
@@ -1,15 +1,35 @@
+import React from 'react';
 import '../App.css';
 import ButtonPanel from './buttonPanel';
 import Display from './display';
 import calculate from '../logic /calculate';
```

Comment:
- [ ] On the line 5 please remove the space in the directory structure as it prevents the app from loading. Simply change this: `import calculate from '../logic /calculate';` to `import calculate from '../logic/calculate';`.